query.js

```diff
@@ -519,21 +522,45 @@ exports.search = (req, res, next) => {
 
 }
 
+/*
+ *  Given an entry's travels array and total number of tours, this function
+ *  returns the sum of all tour times.
+ */
+function getTravelTime(travels, numTours) {
+    
+    var accum = 0;
+    var i;
+    for (i = 1; i <= numTours; i++) {
+        var thisTour = travels.filter(function (d) {
+            return d.tourIndex == i;
+        });
+
+        if (thisTour && thisTour[0]) {
+            var numTravels = thisTour.length;
+            var firstTravel = thisTour[0];
+            var lastTravel = thisTour[numTravels - 1];
```

Comment:
You can inline this and remove numTravels

```suggestion
            var lastTravel = thisTour[thisTour.length - 1];
```